package.json

```diff
@@ -42,9 +42,10 @@
     "flow": "flow",
     "test": "jest --bail --coverage --ci --runInBand && codecov",
     "test:dev": "jest --runInBand --watch",
-    "test:intranetToken": "node ./src/server/utils/getIntranetToken.babel.js",
-    "test:e2e-deps": "node ./e2e/scripts/ensureDrivers.babel.js",
     "test:e2e": "node ./e2e/scripts/runTests.babel.js",
+    "test:e2e-deps": "node ./e2e/scripts/ensureDrivers.babel.js",
+    "test:intranetToken": "node ./src/server/utils/getIntranetToken.babel.js",
+    "test:update": "jest --updateSnapshot --runInBand",
```

Comment:
I've needed this before. Safe to add `test:update`?

Reply:
Ok, I will pull this line.

Reply:
I captured this bit of knowledge for time immemorial: https://app.getguru.com/#/facts/038c60ea-0b72-4a77-a2f3-98dbd4d870f0